src/arrayMethodJoin.js

```diff
@@ -5,8 +5,29 @@
  */
 function applyCustomJoin() {
   [].__proto__.join2 = function(separator) {
-    // write code here
+    let res = '';
+    let separ = separator;
+
+    if (separator === undefined) {
+      separ = ',';
+    }
```

Comment:
```suggestion
    [].__proto__.join2 = function(separator = ',') {
```
can usr this instead of if

Reply:
Agree, set default value for separator in function declaration